src/Template/Pages/Modules/index.twig

```diff
@@ -51,7 +51,13 @@
                             <div class="thumb-cell">{% element 'Modules/thumb' { 'object': object } %}</div>
 
                             {% for prop in properties %}
-                                <div class="{{ prop }}-cell" untitled-label="{{ __('Untitled') }}">{{ Schema.format(object.attributes[prop], schema.properties[prop]) }}</div>
+                                <div class="{{ prop }}-cell" untitled-label="{{ __('Untitled') }}">
+                                    {% if (prop == 'date_ranges') %}
```

Comment:
We need something less hard-coded in order to allow use of `element` to display some attributes in a list.
Just to avoid having a long `{% if ... %} {% elsefi %}..` chain in the future.

For example via a new `Schema.listItem()` method (or similar name) to load a custom element or call `Schema.format()`